docs/installation/devs.rst

```diff
@@ -156,46 +160,62 @@ packages with pip. In the directory where your copy of mrsimulator is, type:
     $ pip install -r requirements-dev.txt
     $ pip install -e .
 
-As always, if you get an error that you don’t have the permission to install the
+As before, if you get an error that you don’t have the permission to install the
 package into the default site-packages directory, you may try installing by adding the
 ``--user`` option.
 
 
 Note for the developers and contributors
 ''''''''''''''''''''''''''''''''''''''''
 
-**Before commits**: Mrsimulator follows python community standards for writing code and
-documentation. To help guide the developers and contributors towards these standards,
-we have created a *.pre-commit-config.yaml* file, that when used with ``pre-commit``, will
-inspect the code and document for issues.
-Type ``pre-commit run`` before git commits to inspect the changes.
+**Before commits**: Mrsimulator follows Python community standards for writing code and documentation.
+To help guide the developers and contributors toward these standards, we have created
+a *.pre-commit-config.yaml* file that, when used with ``pre-commit``, will inspect
+the code and document for issues. To set up ``pre-commit``, type the following one-time
+install statement in the terminals,
+
+.. code-block:: bash
+
+    $ pre-commit install
 
-You can also set up the git hook script to automatically run *pre-commit* on git
-commits with the ``pre-commit install``. Read more about
-`pre-commit <https://pre-commit.com/#3-install-the-git-hook-scripts>`_.
+Once set up, navigate to the root level of the mrsimulator folder and type
 
+.. code-block:: bash
+
+    $ pre-commit run
+
+The above statement auto-fixes some issues and lists others for you to fix. Review the
+changes and address the listed issues before a git commit.
+
+.. You can also set up the git hook script to automatically run *pre-commit* on git commits
+.. with the ``pre-commit install``. Read more about
+.. `pre-commit <https://pre-commit.com/#3-install-the-git-hook-scripts>`_.
+
+.. note::
+    The pre-commit command ignores unstaged changes. Before running ``pre-commit run``, make sure
+    to stage files for a commit.
 
-**Running tests**: For unit tests, we use the pytest module. At the root directory
-of the mrsimulator package folder, type
+**Running tests**: We use the pytest module for unit tests. At the root level
+of the mrsimulator folder, type
 
 .. code-block:: bash
 
     $ pytest
 
 which will run a series of tests.
 
-**Building docs**: We use the sphinx python documentation generator for building docs.
-Navigate to the *docs* folder within the mrsimulator package folder, and type,
+**Building docs**: We use the sphinx Python documentation generator for building docs. Navigate
+to the *docs* folder within the mrsimulator package folder, and type,
 
```

Comment:
@deepanshs Should we add a quick sentence or two about checking code coverage locally? Or is that too in-depth

Reply:
A line should be enough.